mimic/model/rackspace_image_store.py

```diff
@@ -0,0 +1,66 @@
+"""
+An image store representing Rackspace specific images
+"""
+from __future__ import absolute_import, division, unicode_literals
+from characteristic import attributes, Attribute
+from six import iteritems
+from mimic.model.rackspace_images import (RackspaceWindowsImage, RackspaceArchImage,
+                                          RackspaceCentOSPVImage, RackspaceCentOSPVHMImage,
+                                          RackspaceCoreOSImage, RackspaceDebianImage,
+                                          RackspaceFedoraImage, RackspaceFreeBSDImage,
+                                          RackspaceGentooImage, RackspaceOpenSUSEImage,
+                                          RackspaceRedHatPVImage, RackspaceRedHatPVHMImage,
+                                          RackspaceUbuntuPVImage, RackspaceUbuntuPVHMImage,
+                                          RackspaceVyattaImage, RackspaceScientificImage,
+                                          RackspaceOnMetalCentOSImage, RackspaceOnMetalCoreOSImage,
+                                          RackspaceOnMetalDebianImage, RackspaceOnMetalFedoraImage,
+                                          RackspaceOnMetalUbuntuImage)
+
+
+@attributes([Attribute("image_list", default_factory=list)])
+class RackspaceImageStore(object):
+    """
+    A store for images to share between nova_api and glance_api
+    :var image_list: list of Rackspace images
+    """
+    def create_image_store(self, tenant_id):
```

Comment:
This method's documentation is really misleading and I find myself somewhat confused reading it.

It says "generates the data for each image...", but in fact it only does this if the data hasn't already been populated.  Or if some other data were not already attached.